docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: "3"
+
+services:
+
+  database:
+    image: "postgres:11"
+    environment:
+      POSTGRES_USER: omero
+      POSTGRES_DB: omero
+      POSTGRES_PASSWORD: omero
+    networks:
+      - omero
+    volumes:
+      - "database:/var/lib/postgresql/data"
+    command:
+      - "postgres"
+      - "-N"
+      - "500"
+
+  omeroserver:
+    image: "openmicroscopy/omero-server:5.6"
+    environment:
+      CONFIG_omero_db_host: database
+      CONFIG_omero_db_user: omero
+      CONFIG_omero_db_pass: omero
+      CONFIG_omero_db_name: omero
+      ROOTPASS: omero
+    networks:
+      - omero
+    ports:
+      - "4063:4063"
+      - "4064:4064"
+    volumes:
+      - "omero:/OMERO"
+
+  omeroweb:
```

Comment:
You could probably remove this and save the download time?

Reply:
Ack, crud. That wasn't supposed to be included here. I'll push it away, but it's very useful for testing and since it's only downloading once not particularly a problem (unless you mean in travis, and there it's not used at all)